doc/releases/changelog-0.31.0.md

```diff
@@ -4,72 +4,413 @@
 
 <h3>New features since last release</h3>
 
-* Added a new function `qml.ops.functions.bind_new_parameters` that creates a copy of an operator with new parameters without mutating the original operator.
-  [(#4113)](https://github.com/PennyLaneAI/pennylane/pull/4113)
-  [(#4256)](https://github.com/PennyLaneAI/pennylane/pull/4256)
+<h4>Seamlessly create and combine fermionic operators 🔬</h4>
 
-* Added the `TRX` qutrit rotation operation, which allows applying an X rotation on a
-  given subspace.
-  [(#2845)](https://github.com/PennyLaneAI/pennylane/pull/2845)
+* Fermionic operators and arithmetic are now available. 
+  [(#4191)](https://github.com/PennyLaneAI/pennylane/pull/4191)
+  [(#4195)](https://github.com/PennyLaneAI/pennylane/pull/4195)
+  [(#4200)](https://github.com/PennyLaneAI/pennylane/pull/4200)
+  [(#4201)](https://github.com/PennyLaneAI/pennylane/pull/4201)
+  [(#4209)](https://github.com/PennyLaneAI/pennylane/pull/4209)
+  [(#4229)](https://github.com/PennyLaneAI/pennylane/pull/4229)
+  [(#4253)](https://github.com/PennyLaneAI/pennylane/pull/4253)
+  [(#4255)](https://github.com/PennyLaneAI/pennylane/pull/4255)
+  [(#4262)](https://github.com/PennyLaneAI/pennylane/pull/4262)
+  [(#4278)](https://github.com/PennyLaneAI/pennylane/pull/4278)
 
-* Added the `TRY` qutrit rotation operation, which allows applying a Y rotation on a
-  given subspace.
-  [(#2846)](https://github.com/PennyLaneAI/pennylane/pull/2846)
+  There are a couple of ways to create fermionic operators with this new feature:
 
-* Added the `TRZ` qutrit rotation operation, which allows applying a Z rotation on a
-  given subspace.
-  [(#2847)](https://github.com/PennyLaneAI/pennylane/pull/2847)
+  - `qml.FermiC` and `qml.FermiA`: the
+    [fermionic creation](https://docs.pennylane.ai/en/stable/code/api/pennylane.FermiC.html) and
+    [annihilation operators](https://docs.pennylane.ai/en/stable/code/api/pennylane.FermiA.html),
+    respectively. These operators are 
+    defined by passing the index of the orbital that the fermionic operator acts on. For instance, 
+    the operators `a⁺(0)` and `a(3)` are respectively constructed as
```

Comment:
Should not this be in math format, .i.e., :math:`a^\dagger_0 a_3`?

Reply:
We can't get the math to render in both md and rst :cry: